homeassistant/components/google/__init__.py

```diff
@@ -41,17 +42,32 @@
 DEFAULT_CONF_TRACK_NEW = True
 DEFAULT_CONF_OFFSET = '!!'
 
+EVENT_CALENDAR_ID = 'calendar_id'
+EVENT_DESCRIPTION = 'description'
+EVENT_END_CONF = 'end'
+EVENT_END_DATE = 'end_date'
+EVENT_END_DATETIME = 'end_date_time'
+EVENT_IN = 'in'
+EVENT_IN_DAYS = 'days'
+EVENT_IN_WEEKS = 'weeks'
+EVENT_START_CONF = 'start'
+EVENT_START_DATE = 'start_date'
+EVENT_START_DATETIME = 'start_date_time'
+EVENT_SUMMARY = 'summary'
+EVENT_TYPES_CONF = 'event_types'
+
 NOTIFICATION_ID = 'google_calendar_notification'
-NOTIFICATION_TITLE = 'Google Calendar Setup'
+NOTIFICATION_TITLE = "Google Calendar Setup"
 GROUP_NAME_ALL_CALENDARS = "Google Calendar Sensors"
 
 SERVICE_SCAN_CALENDARS = 'scan_for_calendars'
 SERVICE_FOUND_CALENDARS = 'found_calendar'
+SERVICE_ADD_EVENT = 'add_event'
 
 DATA_INDEX = 'google_calendars'
 
 YAML_DEVICES = '{}_calendars.yaml'.format(DOMAIN)
-SCOPES = 'https://www.googleapis.com/auth/calendar.readonly'
+SCOPES = 'https://www.googleapis.com/auth/calendar'
```

Comment:
This is a breaking change in that everyone will be forced to re-authenticate?

Reply:
Yes. You really only need to re-authenticate to use the service though. Everything else will work as before if you don't.